policykit/policyengine/admin.py

```diff
@@ -156,7 +158,8 @@ def save_model(self, request, obj, form, change):
 
 
 class ConstitutionPolicyBundleAdmin(admin.ModelAdmin):
-    fields= ('bundled_policies', 'description')
+    fields= ('bundled_policies',)
+    readonly_fields = ('policy_type',)
```

Comment:
i'm confused - policy_type is not a model field right, so why would it show up in the admin page?